solutions/src/test/java/ru/sberbank/school/task10/ConcurrentFixedThreadPoolTest.java

```diff
@@ -0,0 +1,91 @@
+package ru.sberbank.school.task10;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ConcurrentFixedThreadPoolTest {
+    private static final int COUNT = 3;
+
+    @Test
+    @DisplayName("Наличие исключения, при неправильных аргументах конструктора")
+    void checkIllegalConstructorArgument() {
+        Assertions.assertThrows(IllegalArgumentException.class, () -> new ConcurrentFixedThreadPool(-1));
+        Assertions.assertThrows(IllegalArgumentException.class, () -> new ConcurrentFixedThreadPool(0));;
+    }
+
+    @Test
+    @DisplayName("Наличие исключения, при попытке остановки до старта")
+    void checkCantStoppedBeforeStart() {
+        ConcurrentFixedThreadPool threadPool = new ConcurrentFixedThreadPool(COUNT);
+        Assertions.assertThrows(IllegalStateException.class, threadPool::stopNow);
+    }
+
+    @Test
+    @DisplayName("Наличие исключения, при попытке дважды запустить тердпул")
+    void checkDoubleStart() {
+        ConcurrentFixedThreadPool threadPool = new ConcurrentFixedThreadPool(COUNT);
+        threadPool.start();
+        Assertions.assertThrows(IllegalStateException.class, threadPool::start);
+    }
+
+    @Test
+    @DisplayName("Наличие исключения, при попытке добавить задачу до старта")
+    void cantExecuteBeforeStarted() {
+        ConcurrentFixedThreadPool threadPool = new ConcurrentFixedThreadPool(COUNT);
+        AtomicInteger count = new AtomicInteger(0);
+        Assertions.assertThrows(IllegalStateException.class
+                , () -> threadPool.execute(count::incrementAndGet));
```

Comment:
&#39;,&#39; should be on the previous line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1979235154>*